README.md

```diff
@@ -107,6 +107,18 @@ flash('messages.errors', ['Email is required.', 'Phone is required.']);
 flash('messages.errors'); // Array( 0 => 'Email is required.', 1 => 'Phone is required.' )
 ```
 
+## Flash for current page load only
+
+Sometimes it can be useful to flash a message for the current page load, not the next one. This use case
+arises when a message should be shown in a response to the same request, rather than a redirect.
+
+The `flash()` helper method can be called with an optional **third parameter**, a boolean to toggle whether to flash _now_ instead of in the next page load. The default value of this parameter is `false`: `flash($key, $value = '', $now = false)`.
+
+```php
+flash('message', 'Message for redirect'); // flash on the next page load (default)
+flash('message', 'Message for this response', true); // flash in a response to the same request
+```
```

Comment:
I find this phrasing to be misleading. The flashing is done *now* in both cases. Only with `$now = true` it doesn't persist.
```suggestion
The `flash()` helper method can be called with an optional third parameter, a boolean to toggle whether to keep the flashed variable only for the current request. The default value of this parameter is `false` which will keep the flashed variable for the next request.

```php
flash('message', 'Message for redirect'); // Keep for next request
flash('message', 'Message for this response', true); // Keep only for current request
```
```